### PR TITLE
Add the FAQ in the repository

### LINKED ISSUE #5034: Add info on how to start tracking files already in git
It's helpful to know how to start tracking files already in git using `git lfs`.

The tip in the `README` is only partial (as are several answers on Stack Overflow):
> Tip: if you have large files already in your repository's history, git lfs track will not track them retroactively. To migrate existing large files in your history to use Git LFS, use `git lfs migrate`. 

 I found the answer by @bk2204 [here](https://github.com/git-lfs/git-lfs/issues/874#issuecomment-762937377):

> If you want to track existing files, you need to either use `git lfs migrate import --no-rewrite` with an appropriate pattern or use `git add --renormalize .` after git lfs track (and then commit).

The `git add --renormalize .` made all the difference :+1: thanks! :pray:

### PR DESCRIPTION
**Pull request by bk2204**
Right now, people have to read the FAQ on the wiki, which makes it hard to discover or to use offline.  We want people to have a copy of it installed as part of the documentation for ease of discovery.  Add it to the repository, and add a couple of missing entries to help explain some situations we haven't mentioned yet.

Fixes #5034

### CODE REVIEW

File: docs/man/git-lfs-faq.adoc [hunk 1]
```diff
@@ -0,0 +1,144 @@
+= git-lfs-fae(5)
```
Comment (chrisd8088) [inline docs/man/git-lfs-faq.adoc:1]: Typo: `faq` not `fae`.

File: docs/man/git-lfs-faq.adoc [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+$ cat .gitattributes
+....
+*.bin filter=lfs diff=lfs merge=lfs -text
+*.patch filter=lfs diff=lfstext merge=lfs -text
+....
+----
++
+Be advised that all developers sharing this repo with such a modified
+`.gitattributes` file must have similarly configured the `lfstext` text
+converter, whether globally or on per repository basis.
```
Comment (chrisd8088) [inline docs/man/git-lfs-faq.adoc:115]: I might add an "a" here, as in, "on a per repository basis".

File: docs/man/git-lfs-faq.adoc [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+How do I convert from using Git LFS to a plain Git repository?::
+  If you'd like to stop using Git LFS and switch back to storing your large
+  files in the plain Git format, you can do so with `git lfs migrate export
+  --everything`.  Note that you will need to provide and appropriate `--include`
```
Comment (chrisd8088) [inline docs/man/git-lfs-faq.adoc:120]: Typo: "an" not "and".

File: docs/man/git-lfs-faq.adoc [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+I'm using Git LFS, but I still see GitHub's large file error.  How do I fix this?::
+  GitHub rejects large files anywhere in the history of your repository, not
+  just in the latest commit.  If you're still seeing this message, then you have
+  some large files somewhere in your history, even if in the latest commits
+  you've moved them to Git LFS.
++
+To fix this, you can use `git lfs migrate import --everything` with an
+appropriate `--include` argument.  For example, if you wanted to move your
+`.jpg` and `.png` files into Git LFS, you can do that with
+`git lfs migrate import --everything --include="\*.jpg,*.png"`.  More complicated
```
Comment (chrisd8088) [inline docs/man/git-lfs-faq.adoc:135]: Just out of interest, is there a difference here between `\*` and plain `*`?

**Review by chrisd8088 — APPROVED** [review]
Looks great!  I noted some minor typos only.

File: docs/man/git-lfs-faq.adoc (hunk 4 above)
Comment (bk2204) [reply docs/man/git-lfs-faq.adoc:135]: This is required to make AsciiDoc not consider this bold.  Very fortunately, it's also syntactically correct according to the shell.

File: docs/man/git-lfs-faq.adoc (hunk 1 above)
Comment (bk2204) [reply docs/man/git-lfs-faq.adoc:1]: Ah, yes, this is an open source project, not an RPG campaign.

File: docs/man/git-lfs-faq.adoc [hunk 5]
```diff
@@ -0,0 +1,144 @@
+= git-lfs-faq(5)
```
Comment (chrisd8088) [inline docs/man/git-lfs-faq.adoc:1]: I realize that we should probably put this in section 7 of the manual, actually; that's where Git [puts](https://git-scm.com/docs/git#_guides) the `gitfaq(7)` manual page, for instance.

Sorry not to spot this on a first review.

**Review by chrisd8088 — APPROVED** [review]
Looks great, as I said -- but we should maybe change this to install in `man7` and be `git-lfs-faq(7)`.  That will also mean tweaking a couple of extra `Makefile` [lines](https://github.com/git-lfs/git-lfs/blob/33dcdd663f436c297a7db16c347ca16e5c67d712/Makefile#L700-L701).

File: docs/man/git-lfs-faq.adoc (hunk 5 above)
Comment (bk2204) [reply docs/man/git-lfs-faq.adoc]: No worries.  That's a good thing to note, and I've squashed in a change to that effect.  Once it's green, I'll merge.

**Pull request merged**